src/ttsim/typing.py

```diff
@@ -25,6 +25,7 @@
     QualNamePolicyInputDict = Mapping[str, PolicyInput]
 
     # Specialise from dags' NestedInputDict to GETTSIM's types.
+    NestedInputToSeriesNameDict = Mapping[str, Any | "NestedInputToSeriesNameDict"]
     NestedDataDict = Mapping[str, pd.Series | "NestedDataDict"]
```

Comment:
Just a heads-up that the current type `NestedDataDict` is nonsense (we never use series) and will change via #879.